src/Console/IndexCommand.php

```diff
@@ -89,7 +89,8 @@ public function handle()
 
             if ($bulkQuery->isReady()) {
                 $this->elasticsearchService->bulk($bulkQuery->toArray());
-                $bulkQuery->reset();
+                // Fix memory issue
+                $bulkQuery = new BulkQuery($this->getBulkSize());
```

Comment:
@Jalle19 For some reasons it doesn't work here...

Reply:
Still get `Allowed memory size of 268435456 bytes exhausted (tried to allocate 28672 bytes)`

Reply:
@Jalle19 `$bulkQuery->reset()` works on top also.